src/transformation/visitors/helpers/multi.ts

```diff
@@ -0,0 +1,182 @@
+import * as ts from "typescript";
+import * as lua from "../../../LuaAST";
+import * as utils from "../../utils/helpers";
```

Comment:
`utils` is a bit too broad given how many utils we do have. Maybe `helpers` or `helperUtils`?